sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/rules/QueryExecutionMetering.scala

```diff
@@ -85,10 +85,12 @@ case class QueryExecutionMetering() {
       s"$ruleName $runtimeValue $numRunValue"
     }.mkString("\n", "\n", "")
 
+    val totalTimeInJavaDouble =
+      (totalTime / NANOS_PER_SECOND.toDouble).asInstanceOf[java.lang.Double]
     s"""
        |=== Metrics of Analyzer/Optimizer Rules ===
        |Total number of runs: $totalNumRuns
-       |Total time: ${totalTime / NANOS_PER_SECOND.toDouble} seconds
```

Comment:
How about?
```scala
Total time: ${"%.9f".format(totalTime / NANOS_PER_SECOND.toDouble)} seconds
```

Reply:
> How about?
> 
> ```scala
> Total time: ${"%.9f".format(totalTime / NANOS_PER_SECOND.toDouble)} seconds
> ```

Good idea, String.format() need to use java Double type.

Reply:
maybe we don't need 9 digits after `.`?

Reply:
> maybe we don't need 9 digits after `.`?

If don't care precision, this [advice](https://github.com/apache/spark/pull/25844#discussion_r325989089) is good.